### PR TITLE
Ncnews 13

### PR DESCRIPTION
**Pull request by rangararazin**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -460,4 +460,15 @@ describe("DELETE: /api/comments/:comment_id", () => {
...
   });
+}); //
+
+describe("GET:  /api", () => {
+  test("200: should responds with JSON object with describing all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.endpoints).toBe("object");
```
Comment (callumu21) [inline __tests__/app.test.js]: This test won't tell you enough about the object you are getting back. Think about the information within, keys etc. and try to form a more rigid test, as so far an array would pass here.

File: models/endpoint-model.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const fs = require("fs/promises");
+const path = "./endpoints.json";
+
+exports.readEndpoints = () => {
```
Comment (callumu21) [inline models/endpoint-model.js]: As you aren't dealing with the database, this logic should be in the controller

**Review by callumu21 — CHANGES_REQUESTED** [review]
Just a few comments before this can be approved 👍

File: models/endpoint-model.js (hunk 2 above)
Comment (rangararazin) [reply models/endpoint-model.js]: As not to get confused I am keeping it separate in model.

**Comment by Geo-Beard** [comment]
Looks OK overall - however, your readEndpoints shouldn't have a model - you should just make use of controllers here. Feel free to merge and move on when you're happy with implementation :)

**Pull request merged**